tests/test_migrate_commands.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import json
+
+import mock
+from syncano_cli.config import ACCOUNT_CONFIG
+from syncano_cli.main import cli
+from tests.base import InstanceMixin, IntegrationTest
+
+
+class MigrateCommandsTest(InstanceMixin, IntegrationTest):
+
+    @classmethod
+    def setUpClass(cls):
+        super(MigrateCommandsTest, cls).setUpClass()
+        # invoke login first:
+        cls.runner.invoke(cli, args=['login'], obj={})
+        # and make setup;
+        cls._set_up_configuration()
+
+    @classmethod
+    def _set_up_configuration(cls):
+        SYNCANO_ADMIN_API_KEY = ACCOUNT_CONFIG.get("DEFAULT", "key")
+        if not ACCOUNT_CONFIG.has_section("P2S"):
+            ACCOUNT_CONFIG.add_section("P2S")
+        ACCOUNT_CONFIG.set("P2S", "PARSE_APPLICATION_ID", "xxx")
+        ACCOUNT_CONFIG.set("P2S", "PARSE_MASTER_KEY", "xxx")
+        ACCOUNT_CONFIG.set("P2S", "SYNCANO_INSTANCE_NAME", cls.instance.name)
+        ACCOUNT_CONFIG.set("P2S", "SYNCANO_ADMIN_API_KEY", SYNCANO_ADMIN_API_KEY)
+
+    def test_configure(self):
+        result = self.runner.invoke(cli, args=['migrate', 'configure'], obj={})
+        self.assertIn('PARSE_MASTER_KEY', result.output)
+
+        result = self.runner.invoke(cli, args=['migrate', 'configure', '--force'], obj={})
+        self.assertIn('PARSE_MASTER_KEY', result.output)
+
+        result = self.runner.invoke(cli, args=['migrate', 'configure', '--current'], obj={})
+        self.assertIn('PARSE_MASTER_KEY', result.output)
+
+    def test_parse(self):
+        result = self.runner.invoke(cli, args=['migrate', 'parse'], obj={})
+        self.assertIn('Aborted!', result.output)
+
+    @mock.patch('syncano_cli.parse_to_syncano.parse.connection.ParseConnection.request')
+    @mock.patch('syncano_cli.parse_to_syncano.migrations.transfer.SyncanoTransfer.transfer_objects')
+    @mock.patch('syncano_cli.parse_to_syncano.migrations.transfer.SyncanoTransfer.transfer_files')
+    @mock.patch('syncano_cli.parse_to_syncano.migrations.transfer.SyncanoTransfer.process_relations')
+    def test_class_migrations(self, relations_mock, files_mock, objects_mock, request_mock):
+        relations_mock.return_value = None
```

Comment:
You can define return_value as an argument on `@mock.patch`,
e.g.
`@mock.patch('syncano_cli.parse_to_syncano.migrations.transfer.SyncanoTransfer.process_relations', return_value=None)`
Or if you don't care about asserting .called etc you can do something like that:
`@mock.patch('syncano_cli.parse_to_syncano.migrations.transfer.SyncanoTransfer.process_relations', mock.MagicMock(return_value=None))` - this way it won't be passed as an argument to function